File: dns.lookup/python/dns_lookup_pass.py
try:
    from urllib.request import urlopen, pathname2url
except ImportError:
    from urllib import pathname2url
    from urllib2 import urlopen

domain = 'example.com'
apiKey = 'Your dns lookup api key'
checkType = '_all'

url = 'http://www.whoisxmlapi.com/whoisserver/DNSService?'\
    + 'type=' + pathname2url(checkType)\
    + '&domainName=' + pathname2url(domain)\
    + '&apiKey=' + pathname2url(apiKey)

print(urlopen(url).read().decode('utf8'))

File: dns.lookup/nodejs/dns_lookup_pass.js
var http = require('https');
var querystring = require('querystring');

var domain = 'example.com';
var apiKey = 'Your dns lookup api key';
var checkType = '_all';

var url = 'https://www.whoisxmlapi.com/whoisserver/DNSService';

var params = {
    type: checkType,
    domainName: domain,
    apiKey: apiKey
};

url = url + '?' + querystring.stringify(params);

http.get(url, function(response) {
    var str = '';
    response.on('data', function(chunk) {
        str += chunk;
    });
    response.on('end', function() {
        console.log(str);
    });
}).end();
